src/hooks/useAggregatorStats.ts

```diff
@@ -0,0 +1,58 @@
+import useSWR from 'swr'
+import { ChainId } from '@kyberswap/ks-sdk-core'
+
+type Response = {
+  pools: Record<
+    string,
+    {
+      poolSize: number
+      tvl: number
+      tokenSize: number
+    }
+  >
+}
+
+const chainIdMapping: Partial<Record<ChainId, string>> = {
```

Comment:
you can use   NETWORKS_INFO variable
`NETWORKS_INFO[chainId].route`

Reply:
Thanks. They are all the same, except for BSC. I need `bsc` instead of `bnb` (as in its route), so I decided to manually do mapping like this.

Reply:
can we deal with backend update bsc to bnb =)) ?

Reply:
Ok let me ask

Reply:
BE refused this change request

Reply:
yes.  Name idea 
```
const route = NETWORKS_INFO[chainId].statRoute || NETWORKS_INFO[chainId].route

```
you add only 1 line in NETWORKS_INFO

Reply:
OK. I see you guys really want it to be reusable.
I don't think it is a good way. Let me get this a bit more straight.

1. No where else uses statRoute, so why include it in a global constant? At this moment, this is used only here, so a specific API, for a specific usage and context
2. Is it readable and contextual within this hook to have a dedicated mapping like this? Sure it is
3. Everything stays in this hook. Whatever changes in the NETWORK_INFO never makes this hook fail

How'd you all think?

Reply:
how about this:
`
const  chainIdMapping = {
  [chainId.BSCMAINNET] : 'bsc'
}
const router =  chainIdMapping[chainId] || NETWORK_INFO[chainId].router
`

Reply:
Ok this seems better

Reply:
I think it never changes :D

Reply:
It's more than reusable. If you have a deep search on `NETWORK_INFOS`, there are fields only use once but still lie there.

So why would we put it there? It's the answers for the situation that everytime we think about a piece of data that different accross networks, we first think it must stay in `NETWORK_INFOS`.

Also, might you didn't care about maintainability, when we add a new chain, everything lie in one place is much more easily to find than many piece of data lie everywhere in the repository.

So in your case, what if we gonna add new chain like Celo for example. I fill up every infomation in `constants/networks/celo.ts`, but I missed this object, and this object dont shown up with error. So no one gonna known that I have missed this obj.

Reply:
Valid point. It's bad 👍